### DISCUSSION TITLE
Error While installing Security onion 2.3.61

### DISCUSSION THREAD

**Discussion by faizankhd** [Security-Onion-Solutions/securityonion#5099] [Unsupported Versions]
Hi All

I am getting error while installing Security onion on vmware15

WARNING
installing security onion will destroy all data on this device and partition

My question
Will it damage my drive, since i am installing C drive

Or will it only damage virtual drive

What if i have more virtual machine install

Please suggest

**Comment by fluffyunicorn24** [comment]
Unless you mapped it to the raw disk, which I’m not if that’s a feature on your VMware, you should be fine.

**Reply by faizankhd** [reply]
![image](https://user-images.githubusercontent.com/24580614/128564501-e59755c3-2dd2-4d72-a7d5-0443d5e177b4.png)

This is the config of my vmware

**Reply by faizankhd** [reply]
any suggestion since all my VMware hard disk config is by default i haven't change

I need to protect my data ?

**Reply by dougburks** [reply]
The screenshot you provided shows the settings for "CD/DVD (SATA)" so that doesn't provide any information for this question.

If you click "Hard Disk (SCSI)", it will most likely show that the virtual drive inside the virtual machine is mapped to a file on your physical drive. The default for VMware should be that each virtual machine has its own virtual drive mapped to its own file on the physical drive. 

Assuming all that is true, then when the installer wipes the drive, it will only wipe the virtual drive inside the virtual machine and won't affect your host machine or any other virtual machines.

**Reply by faizankhd** [reply]
Thanks
